JavaScript/openai-assistants-api/src/index.ts

```diff
@@ -0,0 +1,130 @@
+import * as crypto from 'crypto';
+import OpenAI from 'openai';
+import { AutoblocksTracer } from '@autoblocks/client';
+
+const openai = new OpenAI({
+  apiKey: process.env.OPENAI_API_KEY,
+});
+
+const tracer = new AutoblocksTracer(process.env.AUTOBLOCKS_INGESTION_KEY, {
```

Comment:
can we use the new constructor (either set this as apiKey in the object or omit)